includes/class-cli-commands.php

```diff
@@ -134,7 +140,12 @@ public function insert_redirect( $args, $assoc_args ) {
 			$to = esc_url_raw( $args[1] );
 		}
 
-		$redirect = Utilities\insert_redirect( $from, $to, $code );
+		$preserve_parameters = (bool) $assoc_args['preserve'];
+		if ( ! is_bool( $preserve_parameters ) ){
+			WP_CLI::error( sprintf( 'In order to preserve the parameters on a URL, the value must be a boolean, %d is invalid', $preserve_parameters ) );
+		}
```

Comment:
You have cast the variable to a bool so the if test is always going to be false. It is not testing what they entered. 
I think this needs a `%s` here. Otherwise the message is mostly going to say "0 is invalid". It also should use their original parameter (`$assoc_args['preserve']`) in the message, not the modified variable.

Reply:
Thank you for your feedback and catching what I missed, I have made some changes based on your suggestion. Hope you don't mind I will submit a review from you when you have a moment